libs/sdk-ui-kit/styles/scss/header.scss

```diff
@@ -602,3 +607,94 @@ $gd-header-project-width: 230px;
         }
     }
 }
+
+.gd-data-header-menu-section {
+    display: flex;
+    flex: 1 0 auto;
+
+    .icon-sources {
+        border-bottom-right-radius: 0;
+        border-top-right-radius: 0;
+
+        &:hover {
+            border-color: #27b3e0;
+        }
+    }
+
+    .icon-model {
+        border-radius: 0;
+
+        &:hover {
+            border-color: #27b3e0;
```

Comment:
we can reuse the variable defined previously here.

Reply:
I have defined and reuse it thanks